zeeguu/core/account_management/user_account_deletion.py

```diff
@@ -0,0 +1,99 @@
+import sqlalchemy
+from zeeguu.core.model import (
+    User,
+    ArticleDifficultyFeedback,
+    PersonalCopy,
+    SearchFilter,
+    UserPreference,
+    Session,
+)
+
+import time
+import zeeguu.core
+from zeeguu.core.model.starred_article import StarredArticle
+
+db_session = zeeguu.core.model.db.session
+
+from zeeguu.core.model import (
+    SearchSubscription,
+    TopicFilter,
+    TopicSubscription,
+    Teacher,
+    TeacherCohortMap,
+    Session,
+    User,
+    UserActivityData,
+    Bookmark,
+    UserArticle,
+    UserReadingSession,
+    UserExerciseSession,
+)
+from zeeguu.core.model import Article
+
+tables_to_modify = [
```

Comment:
I remember seeing this elsewhere. Ah, yes, in `delete_dev_users`. Would it be possible to remove it from there, and thus, remove the duplication between the two files. I think also the `delete_user_account` method is duplicated with `delete_user`?